apps/mobile/src/test-ids.ts

```diff
@@ -0,0 +1,31 @@
+export const TEST_IDS = {
```

Comment:
In the extension we use enums for test ids, would it be better to have a singular named enum `TestId`?

Reply:
Also, is this file in the best location? 🤔 Should we start a `shared` folder?